sustainability_employee_commuting/tests/test_hr_employee.py

```diff
@@ -0,0 +1,57 @@
+from odoo.tests.common import TransactionCase
+from odoo.tests import tagged
+
+
+@tagged('post_install', '-at_install')
+class EmployeeCommutingTestCase(TransactionCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super(EmployeeCommutingTestCase, cls).setUpClass()
+
+        cls.test_employee = cls.env['hr.employee'].create({
+                'name': 'Test Employee',
+            })
+        
+        cls.commuting_metro = cls.env['carbon.hr.commuting'].create({
+                'carbon_factor_id': cls.env.ref('sustainability.carbon_factor_metro').id,
+                'distance_km': 10,
+                'employee_id': cls.test_employee.id,
+            })
+        cls.commuting_bus = cls.env['carbon.hr.commuting'].create({
+                'carbon_factor_id': cls.env.ref('sustainability.carbon_factor_bus').id,
+                'distance_km': 10,
+                'employee_id': cls.test_employee.id,
+            })
+        cls.commuting_bycicle = cls.env['carbon.hr.commuting'].create({
+                'carbon_factor_id': cls.env.ref('sustainability.carbon_factor_bicycle').id,
+                'distance_km': 10,
+                'employee_id': cls.test_employee.id,
+            })
```

Comment:
```suggestion
        cls.commuting_test_employee = cls.env['hr.employee'].create({
                'name': 'Test Employee',
            })
        
        cls.commuting_commuting_metro = cls.env['carbon.hr.commuting'].create({
                'carbon_factor_id': cls.env.ref('sustainability.carbon_factor_metro').id,
                'distance_km': 10,
                'employee_id': cls.commuting_test_employee.id,
            })
        cls.commuting_commuting_bus = cls.env['carbon.hr.commuting'].create({
                'carbon_factor_id': cls.env.ref('sustainability.carbon_factor_bus').id,
                'distance_km': 10,
                'employee_id': cls.commuting_test_employee.id,
            })
        cls.commuting_commuting_bycicle = cls.env['carbon.hr.commuting'].create({
                'carbon_factor_id': cls.env.ref('sustainability.carbon_factor_bicycle').id,
                'distance_km': 10,
                'employee_id': cls.commuting_test_employee.id,
            })
```
Can you prefix those test variable in order to not overwrite already created one, and future one ?

Reply:
Maybe use Command.create, might be easier to read